intellij/Brandwatch.xml

```diff
@@ -0,0 +1,71 @@
+<code_scheme name="Brandwatch" version="173">
+  <DBN-PSQL>
+    <case-options enabled="false">
+      <option name="KEYWORD_CASE" value="lower" />
+      <option name="FUNCTION_CASE" value="lower" />
+      <option name="PARAMETER_CASE" value="lower" />
+      <option name="DATATYPE_CASE" value="lower" />
+      <option name="OBJECT_CASE" value="preserve" />
+    </case-options>
+    <formatting-settings enabled="false" />
+  </DBN-PSQL>
+  <DBN-SQL>
+    <case-options enabled="false">
+      <option name="KEYWORD_CASE" value="lower" />
+      <option name="FUNCTION_CASE" value="lower" />
+      <option name="PARAMETER_CASE" value="lower" />
+      <option name="DATATYPE_CASE" value="lower" />
+      <option name="OBJECT_CASE" value="preserve" />
+    </case-options>
+    <formatting-settings enabled="false">
+      <option name="STATEMENT_SPACING" value="one_line" />
+      <option name="CLAUSE_CHOP_DOWN" value="chop_down_if_statement_long" />
+      <option name="ITERATION_ELEMENTS_WRAPPING" value="chop_down_if_not_single" />
+    </formatting-settings>
+  </DBN-SQL>
+  <JavaCodeStyleSettings>
+    <option name="CLASS_COUNT_TO_USE_IMPORT_ON_DEMAND" value="100" />
+    <option name="NAMES_COUNT_TO_USE_IMPORT_ON_DEMAND" value="100" />
```

Comment:
Not sure about these tbh, in the other version they're 99 and 5, respectively but I couldn't find out why.

Reply:
What do they do, exactly?

Reply:
I'm not 100% certain, but I think that if there are 100 classes or names, it will change it to a * import 
https://github.com/google/styleguide/pull/447 The google style guide version sets them very high 
https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcards-for-class this was the only thing I could find, so I guess I should set them higher?

Reply:
Yeah I think it's the number of imports (class) and the number of static imports (names) to allow before auto formatting it to a *. These should be as high as possible as the style guide mandates not using the * at all IIRC. 100 is probably fine

Reply:
higher is better- the 9999 you have is perfect.